worker/worker.go

```diff
@@ -186,9 +186,23 @@ func (worker *Worker) createVersion(vcsManager *vcs.Manager, event *api.Event) {
 
 	destDir := vcsManager.GetCloneDir(&event.Service, &event.Version)
 	event.Data["context-dir"] = destDir
-	event.Data["image-name"] = fmt.Sprintf("%s/%s/%s", dockerManager.Registry,
-		strings.ToLower(event.Service.Username), strings.ToLower(event.Service.Name))
-	event.Data["tag-name"] = event.Version.Name
+
+	var imageName, tagName string
+
+	if "" == event.Service.ImageName {
+		imageName = fmt.Sprintf("%s/%s/%s", dockerManager.Registry,
+			strings.ToLower(event.Service.Username), strings.ToLower(event.Service.Name))
+		tagName = fmt.Sprintf("%d-%s", time.Now().Unix(), event.Version.Commit)
```

Comment:
What about if `event.Version.Commit` is empty? 
The format should be `{short commit}-{timestamp}`, like `7654321-170401040120`.

Reply:
done